script.js

```diff
@@ -1,4 +1,495 @@
+//GLOBAL VARIABLES
+//ver 2: player hit or stand
+var DRAW_MODE = "DRAW MODE";
+var RESULT_MODE = "RESULT MODE";
+var GAME_START = "GAME START";
+var PLAYER_CHOICE_MODE = "PLAYER CHOICE MODE";
+var COMPUTER_CHOICE_MODE = "COMPUTER CHOICE MODE";
+var INTRO_MODE = "INTRO MODE";
+var PLACE_BET = "PLACE_BET";
+var HIT = "h";
+var STAND = "s";
+var WIN = "WIN";
+var LOSE = "LOSE";
+var TIE = "TIE";
+
+//IMAGE LIBRARY
+
+var IMAGE_WIN = '<img class = "bottom-img" src ="image/BlackJack_WIN.jpg"/>';
+var IMAGE_LOSE = '<img class = "bottom-img" src ="image/BlackJack_LOSE.jpg"/>';
+var IMAGE_TIE = '<img class = "bottom-img" src ="image/BlackJack_TIE.jpg"/>';
+
+//output is updated by using this function
+var displayHandsStatement = function (playerHand, computerHand) {
+  var playerValue = addAllCards(playerHand);
+  var computerValue = addAllCards(computerHand);
+
+  myOutputValue = `Player Hand has ${listAllPlayerCards(playerHand)}
+  <br><br> Computer Hand has ${listCPUCards(computerHand)}
+  <br><br> Player deck Value is ${playerValue} and CPU total value is ${computerValue}`;
+  //<br><br> Player, would you like to hit or stand? type h for hit and s for stand.;
+
+  return myOutputValue;
+};
+
+//ask player if hit or stand
+var askHitOrStand = function () {
+  myOutputValue = `Player, would you like to hit or stand? type h for hit and s for stand.`;
+  return myOutputValue;
+};
+
+//declares computer wants to stand
+var declareStand = function () {
+  myOutputValue = `Computer decided to stand.`;
+  return myOutputValue;
+};
```

Comment:
The functions here are only creating a string and returning it - maybe just a string variable like `ASK_HIT_OR_STAND = "........."` will suffice.